pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -43,7 +43,7 @@ const (
 	KolideSaasExtensionName = "kolide_grpc"
 
 	// How long to wait before erroring because the osquery process has not started up successfully
-	osqueryStartupTimeout = 1 * time.Minute
+	osqueryStartupTimeout = 20 * time.Second
```

Comment:
might be nice to even link this PR write up here

Reply:
Makes sense, added a comment and updated!